app/components/jobseekers/search_results/job_alerts_link_component.rb

```diff
@@ -6,6 +6,6 @@ def initialize(vacancies_search:, count:, origin:)
   end
 
   def render?
-    @vacancies_search.any? && !ReadOnlyFeature.enabled? && @count.positive?
+    @vacancies_search.any_criteria_given? && !ReadOnlyFeature.enabled? && @count.positive?
```

Comment:
`filters` are slightly different, conceptually, from criteria - I'd say filters excludes keywords (for example) but criteria includes them. Suggest renaming this method to `any_filters?`

Reply:
This method also includes the keyword – I think its meaning is "did the user input anything significant or did they just hit 'Search'"